### DISCUSSION TITLE
Anyway to select last row

### DISCUSSION THREAD

**Discussion by hilmanski** [supabase/supabase#2713] [Questions] [answered]
Hi,
is there any way to retrieve latest row from specific column?

Since if we ordered ascending to false AND limit 1,
it will limit our search first to only one record (which mean only the first one will return). 

Anyone has a success with this?

So I want to get latest row but only select one row , don't return all

**Comment by himeshp** [comment]
use range?

**Comment by cobblestone-fsh** [comment, marked as answer]
This has worked for me:

```sql
select * from countries
  order by id desc
  limit 1
```
